packages/element-web-playwright-common/src/fixtures/user.ts

```diff
@@ -12,6 +12,32 @@ import { sample, uniqueId } from "lodash-es";
 import { test as base } from "./services.js";
 import { Credentials } from "../utils/api.js";
 
+/** Adds an initScript to the given page which will populate localStorage appropriately so that Element will use the given credentials. */
+export async function populateLocalStorageWithCredentials(page: Page, credentials: Credentials) {
```

Comment:
knip is going to be unhappy because you export yet don't use it

Reply:
indeed, have pushed a fix